src/components/breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="flex gap-2">
+    <VaIconMenuCollapsed
```

Comment:
pointer cursor could be nice here, should also be navigateable with keyboard. Maybe use button for this element?

Reply:
hmm, it doesn't seem to make any sense given the current project structure.
We always encounter paths like `category/example`, navigation to `category` is disabled, and navigating to `example` from the current location is redundant(already there).